handler.go

```diff
@@ -6,17 +6,22 @@ import (
 	"reflect"
 )
 
-// Validate the input lambda handler based on source code at
+// validateLambdaHandler validates the lambdaHandler is a valid handler. When the handler is valid, the function will check
```

Comment:
> validate if the input lambdahandler function in valid. If valid, returns whether functions takes context otherwise returns error.
The code of the function is taken from https://github.com/aws/aws-lambda-go/blob/ea03c2814414b2223eff860ed2286a83ed8a195c/lambda/handler.go#L75

Reply:
Updated